.env.rc1

```diff
@@ -13,18 +13,21 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
 
 TESTNET_PROJECT_NAME="celo-testnet-production"
 
-AZURE_EASTUS2_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
-AZURE_EASTUS2_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
-AZURE_EASTUS2_REGION_NAME=eus2
-AZURE_WESTUS_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
-AZURE_WESTUS_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
-AZURE_WESTUS_REGION_NAME=wus
-AZURE_WESTEUROPE_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
-AZURE_WESTEUROPE_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
-AZURE_WESTEUROPE_REGION_NAME=weu
+AZURE_ORACLE_EASTUS2_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_EASTUS2_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_EASTUS2_REGION_NAME=eus2
+AZURE_ORACLE_WESTUS_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_WESTUS_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_WESTUS_REGION_NAME=wus
+AZURE_ORACLE_WESTEUROPE_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
+AZURE_ORACLE_WESTEUROPE_AZURE_TENANT_ID=7cb7628a-e37c-4afb-8332-2029e418980e
+AZURE_ORACLE_WESTEUROPE_REGION_NAME=weu
 AZURE_SOUTHBR_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
```

Comment:
Should these be prefixed with `AZURE_KOMENCI_SOUTHBR` now?

Reply:
Yes, thanks for catching this.